src/i18n/ru/index.js

```diff
@@ -97,59 +97,59 @@ export default {
           label: 'Quantum'
         },
         KeyboardSettings: {
-          label: 'Keyboard Settings'
+          label: 'Настройки клавиатуры'
         },
         AppMediaMouse: {
-          label: 'App, Media and Mouse'
+          label: 'Приложение, медиа и мышь'
         }
       },
       settingsPanel: {
-        title: 'Configurator Settings',
+        title: 'Настройки конфигуратора',
         fastInput: {
           label: 'Fast Input',
           title: 'ctrl + alt + f',
-          help: 'Input keys via keyboard without clicking each position.'
+          help: 'Ввод клавиш через клавиатуру без нажатия на каждую позицию.'
         },
         displaySizes: {
-          label: 'Show Key Sizes',
+          label: 'Показать размеры клавиш',
           title: 'ctrl + alt + u',
-          help: 'Show keycap sizes in Key Units'
+          help: 'Показать размеры клавиш в Key Units'
         },
         toggleTutorial: {
-          label: 'Video Tutorial',
-          title: 'MechMerlin how-to video on Configurator',
-          help: 'MechMerlin video guide'
+          label: 'Видеоурок',
+          title: 'MechMerlin видео с инструкциями о конфигураторе',
+          help: 'MechMerlin видеогид'
         },
         darkmode: {
-          label: 'Toggle Darkmode',
-          title: 'Dark mode'
+          label: 'Переключить в темный режим',
+          title: 'Темный режим'
         },
         language: {
-          title: 'Language'
+          title: 'Язык'
         },
         on: {
-          label: 'On'
+          label: 'Включить'
         },
         off: {
-          label: 'Off'
+          label: 'Выключить'
         }
       },
       errors: {
         invalidQMKKeymap:
-          "Sorry, that doesn't appear to be a valid QMK keymap file.",
+          "Извините, это неверный файл раскладки QMK.",
         kbfirmwareJSONUnsupported:
-          "Sorry, QMK Configurator doesn't support importing kbfirmware JSON files.",
-        unknownJSON: "Sorry, this doesn't appear to be a QMK keymap file.",
-        unsupportedBrowser: "You're using a non supported browser. Please use"
+          "Извините, QMK конфигуратор не поддерживает импорт JSON файлов раскладки прошивки.",
+        unknownJSON: "Извините, это не похоже на файл раскладки QMK.",
+        unsupportedBrowser: "Вы используете не поддерживаемый браузер. Пожалуйста, используйте"
       },
       statsTemplate:
-        '\nLoaded {layers} layers and {count} keycodes. Defined {any} Any key keycodes\n',
+        '\nLoaded {layers} слои и {count} коды клавиш. Определен {any} любой код кодов клавиш\n',
```

Comment:
`Loaded` -> `Загружено`. В фигурных скобках - это количество. Так что, наверное, `Загружено X слоев и Y кодов клавиш.` А вот про `{any} Any key keycodes` надо спросить, думаю. Мне кажется, это что-то типа Z каких-то ключевых клавиш.


![image](https://user-images.githubusercontent.com/25141164/66419164-3b7bff80-ea0c-11e9-91ad-4ae19381e024.png)

Reply:
@yanfali, could you please give me a little bit more context for the phrase "Defined {any} Any key keycodes\n'? What do you mean with "Any" which is capitalized?

Reply:
@nabokovas,

I'm sure @yanfali will correct me if necessary, but:

There are ways to define keycodes on the qmk_firmware side such that QMK Configurator can compile them even though they aren't built into QMK Firmware's core like, say, your alphabetical, function or numpad keycodes are.

The `Any` class is basically for keys of this nature that aren't otherwise classed, like a key that changes your layer or toggles backlight would be.

Reply:
It's basically metadata about JSON files you load up containing keymaps. It's really more for debugging, I would leave it as literal as possible